.github/workflows/ci_BE.yml

```
@@ -2,7 +2,7 @@ name: Backend Unit Tests
 
 on:
   pull_request:
-    branches: [dev, main]
+    branches: [dev, main, CIRCLES-331/user-data-to-backend]
```

This (and other occurrences) need to be removed
```suggestion
    branches: [dev, main]
```